src/Entity/View.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\ViewRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: ViewRepository::class)]
+class View
```

Comment:
Should extend https://github.com/itk-dev/economics/blob/develop/src/Entity/AbstractBaseEntity.php
It will handle id, Blamable, Timestampable